packages/injected/README.md

```diff
@@ -90,6 +90,7 @@ console.log(connectedWallets)
 - BifrostWallet - _Desktop & Mobile_
 - Safeheron - _Desktop_
 - Talisman - _Desktop_
+- Coin98 Wallet - _Desktop & Mobile_
```

Comment:
```suggestion
- Ronin Wallet - _Desktop & Mobile_
- Coin98 Wallet - _Desktop & Mobile_
- SubWallet - _Desktop & Mobile_
```

These were missing in the README but are in the docs site

Reply:
I merged latest docs just now how's that?